data-prepper-plugins/key-value-processor/README.md

```diff
@@ -45,6 +45,11 @@ When run, the processor will parse the message into the following output:
 * `exclude_keys` - An array specifying the parsed keys which should not be added to the event. By default no keys will be excluded.
   * Default: `[]`
   * Example: `exclude_keys` is `["key2"]`. `key1=value1&key2=value2` will parse into `{"key1": "value1"}`
+* `default_keys` - A hash specifying the default keys and their values which should be added to the event in case these keys do not exist in the source field being parsed.
```

Comment:
I think we should rename this. Data Prepper uses the term "keys" to refer to key, not both key and value. So it seems this should be a list of just key names. I think `default_values` may be a better name. It could also be `defaults` if we prefer. But, I think `default_values` is more explicit in terms of what is being defaulted.

Reply:
maybe we can rename to `default_pairs`?